src/ring_theory/jacobson_ideal.lean

```diff
@@ -41,12 +41,14 @@ Jacobson, Jacobson radical, Local Ideal
 universes u v
 
 namespace ideal
-variables {R : Type u} [comm_ring R] {I : ideal R}
-variables {S : Type v} [comm_ring S]
+variables {R : Type u} {S : Type v}
 open_locale polynomial
 
 section jacobson
 
+section ring
+variables [ring R] [ring S] {I : ideal R}
+
 /-- The Jacobson radical of `I` is the infimum of all maximal ideals containing `I`. -/
```

Comment:
```suggestion
/-- The Jacobson radical of `I` is the infimum of all maximal (left) ideals containing `I`. -/
```
(I think when generalising to noncommutative rings it's import to state what sort of ideals we're talking about, even if when you know mathlib well enough you know they are all left ideals! :-)

Reply:
Good idea, done